lib/convection/model/template/resource/aws_auto_scaling_auto_scaling_group.rb

```diff
@@ -30,6 +30,51 @@ class AutoScalingGroup < Resource
           def render(*args)
             super.tap do |resource|
               render_tags(resource)
+              @update_policy.render(resource) unless @update_policy.nil?
+            end
+          end
+
+          def update_policy(&block)
+            @update_policy = UpdatePolicy.new
+            @update_policy.instance_exec(&block)
+          end
+
+          ##
+          # UpdatePolicy for AWS::AutoScaling::AutoScalingGroup
+          ##
+          class UpdatePolicy
```

Comment:
Might what to create a separate dir for this, it is a resource attribute not a resource property.  I think the pattern still applies though.  See this [PR](https://github.com/rapid7/convection/pull/46) for an example.